quiz.css

```diff
@@ -0,0 +1,80 @@
+/*Validated using css validator, no errors found*/
+
+/*overall styling*/
+* {
+	box-sizing: border-box;
+}
+
+.main-content {
+	max-width: 1020px;  
+	margin: 0 auto;
+	font-family: Arial, Helvetica, sans-serif;
+}
+
+/*card styling*/
+
+article.card {
+	float: left; /*v2: change this area using media queries for mobile-responsive layout. currently this layout only works on screens about 720px and wider.*/
+	width: 30%;
+	border: 1px solid lightblue;
+	margin: 5px 1.666666667%;
+}
+
+article > header { 
+	border: 1px dotted lightgrey;
+	text-align: center;
+	font-size: .8em;
+}
+
+/*SECTION 1 (information) STYLING*/
+.product-image {
+	display: block;
+	margin: 15px auto;
+	width: 150px;
+	height: 150px; /*updated to have image sizes in CSS not HTML per comment*/
+}
+
+
+div.product-description {
+	height: 190px;
+}
+
+div.product-description p {
+  font-size: 0.85em;
+  padding: 10px;
+  text-align: justify;
+}
+
+.product-availability {
+	background-color: darkgrey;
+	color: yellow;
+	width: 100%;
+	margin: 8px 0;
+	font-size: .8em;
+	line-height: 1.5em;
+}
+
+/*sections 2 and 3 styling. V2 refactor to be one section? or at least have the same class? */
+
+.specifications h3, .pricing h3 {
+	font-size: .9em;
+	font-weight: normal;
+	margin: 10px 0 5px 5px;
+}
+
+.specifications ul, .pricing ul {
+	padding: 0 0 5px 10px;
+	margin: 0;
+}
+
+.specifications ul li, .pricing ul li {
+	font-size: .8em;
+	line-height: 1em;
+	font-weight: bold;
+	list-style-type: none;
+}
+
+.specifications p {
+	font-size: .6em;
+	margin: 0 0 0 2px; /*needed to include these 0s to override standard p margin width */
+}
```

Comment:
Like Github is helpfully indicating, it's standard practice to have a blank line at the end of your files.  This is a holdover from back in the day you can read more about [here](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) if you so desire.